src/sandbox.rkt

```diff
@@ -129,14 +129,14 @@
 
 (define (dummy-table-row result status link)
   (define test (test-result-test result))
+  (define repr (get-representation (test-output-prec test)))
```

Comment:
Honestly we should provide a wrapper around `test-output-prec` which extracts the representation.